### ISSUE TITLE
Cancellation failed in browser

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pylance-release#5420]
1. go to `vscode.dev`
2. create python file
3. type `from typing import ClassVar`
4. do `find all reference` on `ClassVar`

Expect: no error
Actual:
```
[Error - 2:22:11 AM] Request textDocument/references failed.
  Message: Request textDocument/references failed with message: Cancellation over SharedArrayBuffer doesn't support cancellation events
  Code: -32603 
```

**Comment by heejaechang** [comment]
no regression.

**Comment by rchiodo** [comment]
We must be listening for a cancelled event somewhere. That's what causes that error to show up. Likely some new code added a listener.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2024.1.103, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20241103-31-January-2024-PreRelease)
